spot_rl_experiments/spot_rl/envs/nav_env.py

```diff
@@ -207,7 +208,20 @@ def get_success(self, observations):
         return succ
 
     def get_observations(self):
-        return self.get_nav_observation(self.goal_xy, self.goal_heading)
+        # Update the point goal given depth and rgb images
+        if self._enable_nav_goal_change:
+            arm_depth, arm_depth_bbox = self.get_gripper_images()
```

Comment:
The depth returned by this method is "scaled". So we cannot use this method as is.

We need to add a new publisher for publishing "unscaled" / "raw" depth as obtained from BD (Check `SpotLocalRawImagesPublisher` in https://github.com/facebookresearch/spot-sim2real/tree/experiment/3d_object_detection_with_spotcamera which adds that additional publisher)

We need unscaled depth to get true reading of depth as a pixel coordinate (u,v)